tikv-configuration-file.md

```diff
@@ -728,27 +731,29 @@ Configuration items related to Raftstore.
 
 ### `apply-max-batch-size`
 
-+ The maximum number of requests for data flushing in one batch
++ Raft state machines process data write requests in batches by the BatchSystem. This configuration item specifies the maximum number of Raft state machines that can execute the requests in one batch.
 + Default value: `256`
 + Minimum value: greater than `0`
++ Maximum value: `10240`
 
 ### `apply-pool-size`
 
 + The allowable number of threads in the pool that flushes data to storage, which is the size of the Apply thread pool. When you modify the size of this thread pool, refer to [Performance tuning for TiKV thread pools](/tune-tikv-thread-performance.md#performance-tuning-for-tikv-thread-pools).
 + Default value: `2`
-+ Minimum value: greater than `0`
++ Value ranges: `[1, CPU * 10]` (`CPU` means the number of your CPU cores)
 
 ### `store-max-batch-size`
```

Comment:
This part was updated in [#8644](https://github.com/pingcap/docs-cn/pull/8644) and revised in [#8784](https://github.com/pingcap/docs-cn/pull/8784)